### PR TITLE
Mention /tests/index.html in Testing Introduction

### PR DESCRIPTION
**Pull request by jsncmgs1**
I thought it might be worth mentioning that the `/tests` URI renders the `tests/index.html` template, in case someone need to customize it, or add some 3rd party code (test google maps or something similar without stubbing the API?)

I didn't see the file mentioned anywhere else in the docs, I apologize if I missed it!

### CODE REVIEW

File: source/testing/index.md [hunk 1]
```diff
@@ -67,7 +67,8 @@ Run your tests with `ember test` on the command-line. You can re-run your tests
 file-change with `ember test --server`.
 
 Tests can also be executed when you are running a local development server (started by
-running `ember server`), at the `/tests` URI.  A word of caution using this approach:
+running `ember server`), at the `/tests` URI.  This renders the `tests/index.html` template.
```
Comment (rwjblue) [inline source/testing/index.md]: Perhaps a slight tweak to the flow:

> at the `/tests` URI which renders the `tests/index.html` template.

What do you think?

**Comment by jsncmgs1** [comment]
I think your version reads much better.

File: source/testing/index.md (hunk 1 above)
Comment (jsncmgs1) [inline source/testing/index.md]: I think your version reads much better.

**Comment by locks** [comment]
Squashy squash please!

**Comment by jsncmgs1** [comment]
My bad. New to contributing :)

**Comment by locks** [comment]
No problemo, thanks for contributing :)

**Pull request merged**